packages/xo-web/src/common/xo/index.js

```diff
@@ -965,28 +968,44 @@ export const startVm = async (vm, host) => {
 
   const id = resolveId(vm)
   const hostId = resolveId(host)
-  return _call('vm.start', {
-    id,
-    host: hostId,
-  }).catch(async reason => {
-    if (!forbiddenOperation.is(reason)) {
+  return _startVm(id, hostId).catch(async reason => {
+    if (!forbiddenOperation.is(reason) && reason.data.code !== 'DUPLICATED_MAC_ADDRESS') {
       throw reason
     }
 
-    const choice = await chooseActionToUnblockForbiddenStartVm({
-      body: _('blockedStartVmModalMessage'),
-      title: _('forceStartVmModalTitle'),
-    })
+    if (reason.data.code === 'DUPLICATED_MAC_ADDRESS') {
+      // Retry without checking MAC addresses
+      try {
+        await confirm({
+          title: _('forceStartVm'),
+          body: _('vmWithDuplicatedMacAddressesMessage'),
+        })
+        await _startVm(id, hostId, { bypassMacAddressesCheck: true })
+      } catch (error) {
+        if (error === undefined) {
+          return
```

Comment:
I think we need to rethrow `undefined` here so it can be handled as "the modal has been closed".